### DISCUSSION TITLE
Three files will appear after packaging and publishing. Is there any way to package the other two dynamic libraries into the program?

### DISCUSSION THREAD

**Discussion by Seikaijyu** [AvaloniaUI/Avalonia#8356] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68497295/174294047-484c71d3-b981-4a12-93cc-2c1fd908149b.png)
I hope to embed the dynamic link library into the program at the time of release. Can I do that? What should I do?

**Comment by jp2masa** [comment, marked as answer]
https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file/overview

In particular, [Including native libraries](https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file/overview#including-native-libraries).
